src/euroeval/dataset_configs/estonian.py

```diff
@@ -0,0 +1,18 @@
+"""All Estonian dataset configurations used in EuroEval."""
+
+from ..data_models import DatasetConfig
+from ..languages import ET
+from ..tasks import SENT
+
+### Unofficial datasets ###
+
+ESTONIAN_VALENCE_CONFIG = DatasetConfig(
+    name="estonian-valence",
+    pretty_name="the Estonian valence corpus reorganized for EuroEval",
```

Comment:
Another naming consistency.


```suggestion
    pretty_name="the Estonian sentiment classification dataset Estonian Valence",
```